plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -105,10 +102,10 @@
 
     <!-- A "Connector" represents an endpoint by which requests are received
          and responses are returned. Documentation at :
-         Java HTTP Connector: /docs/config/http.html (blocking & non-blocking)
+         Java HTTP Connector: /docs/config/http.html
          Java AJP  Connector: /docs/config/ajp.html
          APR (HTTP/AJP) Connector: /docs/apr.html
-         Define a non-SSL HTTP/1.1 Connector on port 8080
+         Define a non-SSL/TLS HTTP/1.1 Connector on port 8080
     -->
     <Connector address="0.0.0.0" port="8080" protocol="org.apache.coyote.http11.Http11NioProtocol"
```

Comment:
This is an old post from 2015, probably related to tomcat 8.0. But thinking on it again, I agree with you, we should only upgrade tomcat in a first time and made additional changes in a second time when we will be sure everything is ok with tomcat 8.5